src/ResourceManagement/OperationalInsights/OperationalInsightsManagement/Generated/ILinkedServiceOperations.cs

```diff
@@ -0,0 +1,131 @@
+// 
+// Copyright (c) Microsoft and contributors.  All rights reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//   http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// 
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// 
+
+// Warning: This code was generated by a tool.
+// 
+// Changes to this file may cause incorrect behavior and will be lost if the
+// code is regenerated.
+
+using System;
```

Comment:
Please reconsider your directory names. We are already hitting multiple cases where CI is failing due to Windows_Max_Path limitation.
Where the entire file path exceeds 260 character, msbuild will fail.
for e.g. rather than making your directory structure as src/ResourceManagement/OperationalInsights/OperationalInsightsManagement/Generated
how about naming it as
src/ResourceManagement/OperationalInsights/Management/Generated